sdk/python/feast/templates/aws/feature_store.yaml

```diff
@@ -0,0 +1,3 @@
+project: my_project
+registry: s3://data/registry.db
```

Comment:
For now let's use a local registry in the template, since the bucket here is hardcoded and we haven't thought through the user experience on this.